apps/dot-voting/app/index.html

```diff
@@ -7,7 +7,7 @@
 
 <body>
 
-    <div id="root"></div>
+    <div id="dotvoting"></div>
```

Comment:
Do we need to rename this?

Reply:
I cleaned up, made all standard. Not sure if there is any argument for or against? In any case sorry for the commit noise!